### PR TITLE
Add files via upload

### LINKED ISSUE #2362: Add text to readme to encourage stars.
### Overview
We should update the README so that we can encourage more people to star our repo so that we can get noticed.

### Action Items
- [ ] Add the following text to the bottom of the website's readme file

```
## We love our Stars ⭐⭐⭐
Thanks to the following people who have given us a star on our repo: 
[![Stargazers repo roster for @hackforla/website](https://reporoster.com/stars/hackforla/website)](https://github.com/hackforla/website/stargazers)
```

### Resources/Instructions
This code comes from this: https://github.com/nastyox/Repo-Roster
location of file to edit in your IDE: **gh-pages/README.md**

### PR DESCRIPTION
**Pull request by chrisfoose**
Fixes #2362

### What changes did you make and why did you make them ?

  -  Added requested text to bottom of the document.
  -  No screenshot.
  -

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

image

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
image

</details>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+5. Agendas
+6. Project Terminology
+
+This is a standard [Jekyll](https://jekyllrb.com/) site hosted right here on [GitHub pages](https://pages.github.com/).
+
+## Contributing
+
+Once you have gone through onboarding, go to the [WIKI](https://github.com/hackforla/website/wiki). For developers looking for the standard README, see the [CONTRIBUTING](./CONTRIBUTING.md) document for more details on how to get started. Also see our [Code of Conduct](./CODEOFCONDUCT.md) and [License](./LICENSE.txt) documents.
+
+## We love our Stars ⭐⭐⭐
```
Comment (glenflorendo) [inline README.md]: nit. I believe headings should be surrounded by blank lines, similar to how `##Contributing` (line 16) has newlines before and after it (line 15 and line 17).

Comment (glenflorendo) [reply README.md]: nit. Just wondering if we should capitalize all of the words here, since it's a heading title. 🤔

**Review by glenflorendo — CHANGES_REQUESTED** [review]
Really great work! I especially like the image. 🤗

Just small nit picks, and you're good to go!

**Review by adrian-zaragoza — DISMISSED** [review]
This looks good to me! Thank you @chrisfoose !

File: README.md [hunk 2]
```diff
@@ -1,18 +1,23 @@
...
+1. Project Definition
+2. Joining the Team
+3. Tools by Role
+4. Resources by Role
+5. Agendas
+6. Project Terminology
+
+This is a standard [Jekyll](https://jekyllrb.com/) site hosted right here on [GitHub pages](https://pages.github.com/).
+
+## CONTRIBUTING
```
Comment (glenflorendo) [inline README.md]: nit. Oh, sorry! I didn't mean all caps / uppercase. I meant like "Contributing".

File: README.md [hunk 3]
```diff
@@ -1,18 +1,23 @@
...
+
+Once you have gone through onboarding, go to the [WIKI](https://github.com/hackforla/website/wiki). For developers looking for the standard README, see the [CONTRIBUTING](./CONTRIBUTING.md) document for more details on how to get started. Also see our [Code of Conduct](./CODEOFCONDUCT.md) and [License](./LICENSE.txt) documents.
+
+## WE LOVE OUR STARS ⭐⭐⭐
```
Comment (glenflorendo) [inline README.md]: nit. Same as above. I meant like "We Love Our Stars ⭐⭐⭐". Sorry for being unclear!

**Review by glenflorendo — CHANGES_REQUESTED** [review]
Great work on adding the spacing, and I'm really sorry for being unclear about the capitalization previously.

nit. Could you please update the headings for "Contributing" and "We Love Our Stars ⭐⭐⭐".

**Review by glenflorendo — APPROVED** [review]
@chrisfoose Awesome work!

**Review by macho-catt — APPROVED** [review]
LGTM

**Pull request merged**